store/tools/ics23/go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/cosmos/cosmos-sdk v0.46.0-rc3
 	github.com/cosmos/iavl v0.18.0
 	github.com/lazyledger/smt v0.2.1-0.20210709230900-03ea40719554
-	github.com/tendermint/tendermint v0.34.20-rc1
+	github.com/tendermint/tendermint v0.35.2
```

Comment:
This change seems unrelated to the issue

Reply:
that's really wierd.  

I'll revert.  Thanks for the review.